texttospeech/list_voices/list_voices.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+/*
+Example usage:
+	go get -u cloud.google.com/go/texttospeech/apiv1
+	go run list_voices.go
+*/
+
+import (
+	"fmt"
+	"log"
+
+	"golang.org/x/net/context"
+
+	texttospeech "cloud.google.com/go/texttospeech/apiv1"
+	texttospeechpb "google.golang.org/genproto/googleapis/cloud/texttospeech/v1"
+)
+
+// [START tts_list_voices]
+func ListVoices() error {
+	ctx := context.Background()
+
+	client, err := texttospeech.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	// Performs the list voices request
+	request := texttospeechpb.ListVoicesRequest{}
+	response, err := client.ListVoices(ctx, &request)
+	if err != nil {
```

Comment:
(everywhere)

prefer compact style:

`if resp, err := ...; err != nil {`

I'd also inline the `request` variable here.

Reply:
Changed